src/tqec/plaquette/plaquette.py

```diff
@@ -208,6 +264,62 @@ def without_plaquettes(self, indices: Collection[int]) -> Plaquettes:
     def items(self) -> Iterable[tuple[int, Plaquette]]:
         return self.collection.items()
 
+    def to_dict(
+        self, plaquettes_to_indices: dict[Plaquette, int] | None = None
+    ) -> dict[str, Any]:
+        """Return a dictionary representation of the plaquettes.
+
+        The dictionary is intended to be used as a JSON object.
+        """
+
+        def convert(value: Plaquette) -> Any:
+            return (
+                plaquettes_to_indices[value]
+                if plaquettes_to_indices
+                else value.to_dict()
+            )
```

Comment:
Why do you need that? Is there cases where `plaquettes_to_indices` does not contain all the plaquettes that might be explored?

Reply:
Just got it, disregard my above comment